react-props/react-props_button/src/App.js

```diff
@@ -1,6 +1,33 @@
 import React from "react";
 import "./styles.css";
 
+function Button({ color, disabled, text, onClick }) {
+  return (
+    <button
+      disabled={disabled}
```

Comment:
Also, not sure if this works. For me it didn't, I had to put a boolean: disabled = {true} or disabled = {false}. So check, if your button really is disabled with your code.